cmd/scan.go

```diff
@@ -1,16 +1,16 @@
 package cmd
 
 import (
-	"vulnbot/internal"
+	"github.com/underdog-tech/vulnbot/internal"
 
 	"github.com/spf13/cobra"
 )
 
 // scanCmd represents the scan command
 var scanCmd = &cobra.Command{
-	Use:   "scan",
-	Short: "",
-	Long: ``,
+	Use:     "scan",
```

Comment:
q: why the spaces here?

Reply:
Those were automatically added by `gofmt` 😄 We probably need to add some kind of pre-commit config for that.

Reply:
gotcha, thanks!